samples/consumer-driven-contracts/Core_9/Consumer2/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.ConsumerDrivenContracts.Consumer2";
+        var endpointConfiguration = new EndpointConfiguration("Samples.ConsumerDrivenContracts.Consumer2");
+        endpointConfiguration.UsePersistence<NonDurablePersistence>();
+        var transport = endpointConfiguration.UseTransport(new LearningTransport());
+        endpointConfiguration.UseSerialization<XmlSerializer>();
+
+        // uncomment below to demonstrate json
+        //endpointConfiguration.UseSerialization<NewtonsoftJsonSerializer>();
```

Comment:
Did you check if this work on the system json serializer?

Reply:
I went back and took a closer look at this sample, and it seems like there are some outdated assumption/requirements in here. I was able to get it to work with all of the serializers without   any special configuration or behaviors. I went ahead and revised all versions of the samples and the docs page for it.